website/pages/en/sunrise.mdx

```diff
@@ -12,7 +12,7 @@ This plan draws on many previous developments from The Graph ecosystem, includin
 
 ### What are the phases of the sunrise of decentralized data?
 
-**Sunray**: Enable support for hosted service chains, offer a seamless upgrade flow, offer a free plan on The Graph Network.  
+**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and simple payment options.  
```

Comment:
Nitpick but the last item in the enumeration is missing a verb so what do you think about:

```suggestion
**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network as well as simple payment options.  
```